Mapper/BaseGroupedMapper.php

```diff
@@ -256,6 +262,16 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * TODO: make this method abstract before releasing 4.0
+     *
+     * @return string
+     */
+    /*abstract*/ protected function getName()
+    {
+        return 'default';
```

Comment:
I just added a deprecation warning.

This method is used to build the translation id : ``form.group_foo``, ``show.group_foo`` or ``default.group_foo``.

If I return nothing, it will only give ``group_foo``.